bin/varnishd/cache/cache_vrt.c

```diff
@@ -590,6 +590,61 @@ VRT_UpperLowerStrands(VRT_CTX, VCL_STRANDS s, int up)
 	return (r);
 }
 
+// RFC7232, 3.2 without obsolete line folding:
+// ASCII VCHAR + TAB + obs-text (0x80-ff)
+static inline VCL_BOOL
+validhdr_vct(const char *p)
+{
+	AN(p);
+	for(;*p != '\0'; p++)
+		if (vct_isctl(*p) && !vct_issp(*p))
```

Comment:
Didn't we agree that we should have `VCT_OBSTEXT` and `vct_ishdrval()` for the bench?

Reply:
I have pushed a variant with a separate bit in the vct table. I am not sure which other option would make sense?